samples/java-customer-registry/src/main/java/customer/CustomerValueEntityHandler.java

```diff
@@ -56,36 +54,43 @@
   }
 
   @Override
-  public ValueEntityBase.Effect<Any> handleCommand(
+  public ValueEntityBase.Effect<? extends GeneratedMessageV3> handleCommand(
       Any command, Any state, CommandContext<Any> context) throws Throwable {
+
```

Comment:
this is a leftover from #73

Reply:
I think this should be just Any/Object, why should it be GeneratedMessageV3?

Reply:
Because the type that is returned from the generate code is the user defined type, which is a `GeneratedMessageV3`, not an `Any`.

Reply:
As far as I understand they could return anything though, for example some class annotated with @jsonable and the serialization will handle that. Requiring it to be a generated message disallows that?